plugins/nf-ga4gh/src/main/nextflow/ga4gh/tes/executor/TesTaskHandler.groovy

```diff
@@ -234,11 +238,12 @@ class TesTaskHandler extends TaskHandler {
         return size != null ? ((double)size.bytes)/1073741824 : null
     }
 
-    private TesInput inItem( Path realPath, String fileName = null) {
+    private TesInput inItem( Path realPath, String fileName = null, Boolean isBin = false) {
         def result = new TesInput()
         result.url = realPath.toUriString()
         result.path = fileName ? "$WORK_DIR/$fileName" : "$WORK_DIR/${realPath.getName()}"
         log.trace "[TES] Adding INPUT file: $result"
+        result.path = isBin ? realPath : result.path
```

Comment:
This will result in the bin directory being mounted in the container at the same path as Nextflow's main working directory.

All other "non-bin" TES inputs will be mounted at /work by default in the container.